packages/docusaurus-theme-classic/codeTranslations/he.json

```diff
@@ -0,0 +1,55 @@
+{
+  "theme.AnnouncementBar.closeButtonAriaLabel": "לסגור",
+  "theme.CodeBlock.copied": "הועתק",
+  "theme.CodeBlock.copy": "להעתיק",
+  "theme.CodeBlock.copyButtonAriaLabel": "להעתיק קוד",
+  "theme.NotFound.p1": "אנחנו לא מוצאים מה שאתה מנסה לחפש.",
+  "theme.NotFound.p2": "נא פנה למנהל האתר מאיפה קיבלת קישור הזה ותגיד לו שקישור לא תקין.",
+  "theme.NotFound.title": "דף לא נמצא",
+  "theme.Playground.liveEditor": "Live Editor",
+  "theme.Playground.result": "תוצאה",
+  "theme.PwaReloadPopup.closeButtonAriaLabel": "לסגור",
+  "theme.PwaReloadPopup.info": "קיימת גרסא חדשה",
+  "theme.PwaReloadPopup.refreshButtonText": "לרענן",
+  "theme.SearchBar.label": "חיפוש",
+  "theme.SearchPage.algoliaLabel": "חיפוש by Algolia",
+  "theme.SearchPage.documentsFound.plurals": "נמצא מסמך אחד|{count} מסמכים נמצאו",
```

Comment:
"he" has 4 plural forms, but this pluralized string only handle 2 plural forms

Reply:
@slorber these translations are correct, nothing is missing as far as I can tell.

Reply:
@guyca thanks

Can you let me know in which "plural category" are these 2 sentences then?
נמצא מסמך אחד
{count} מסמכים נמצאו

Reply:
נמצא מסמך אחד - ONE
{count} מסמכים נמצאו - MANY

Reply:
isn't there a plural form for "two" here, like in the other cases?

Reply:
No - only one and many (other) forms

Reply:
As I need to declare one,two,many,other, I used:

```
  "theme.SearchPage.documentsFound.plurals": "נמצא מסמך אחד|{count} מסמכים נמצאו|{count} מסמכים נמצאו|{count} מסמכים נמצאו"
```

Please let me know if there's a fix to make.
https://github.com/facebook/docusaurus/pull/4811

Reply:
נמצא מסמך אחד - ONE
נמצאו שני מסמכים - Two
{count} מסמכים נמצאו - MANY (OTHER)

Notice that the difference between two and many are just writing the count 2 in Hebrew. You can use the many-form instead because the two-form is not special

Reply:
too bad, seen your comments after merging :D 

I understand the distinction is like "2 documents" vs "two documents", just wonder which is the more natural? If Hebrew has a distinct plural form for 2 and not English, there must be a good reason :)

Reply:
Hebrew has some words that have a form for two (like in the table example you added) 
שנה year
שנתיים 2 years
שנים some years

Most of them are written with יים suffix

מסמך is not one of them

https://he.m.wikipedia.org/wiki/זוגי_(בלשנות)

Reply:
So it seems like your solution is better than writing the word 2 in hebrew

Reply:
Thanks, I understand :) yes, we can use the same text multiple times for all plural forms if it make sense like in this case